generators/opa5/templates/Journey.ts

```diff
@@ -0,0 +1,28 @@
+import Opa5 from "sap/ui/test/Opa5";
+import opaTest from "sap/ui/test/opaQunit";
+import <%= viewName %>Page from "./pages/<%= viewName %>";
+
+const onThe<%= viewName %>Page = new <%= viewName %>Page();
+
+Opa5.extendConfig({
+	viewNamespace: "<%= uimoduleName %>.view",
+	autoWait: true
+});
+
+QUnit.module("<%= viewName %>");
+
+opaTest("Should have correct title", function() {
+	// Arrangements
+	onThe<%= viewName %>Page.iStartMyUIComponent({
```

Comment:
`void onThe<%=`

Reply:
Basically I agree that `iStartMyUIComponent` and `iTeardownMyApp` return `jQuery.Promise` so I  it should be properly taken care of (and I don't know why eslint@typescript-eslint/no-floating-promises did not complain about that; afaik this rule is [part of eslint-plugin-fiori-tools](https://github.com/SAP/open-ux-tools/blob/main/packages/eslint-plugin-fiori-tools/eslintrc-typescript.js#L6) 🤷🏻).
But I'm curious. What's the benefit of using using `void` as it seems to run just fine without it?

Fun Fact: As soon as I add `void` I get a complaint from SonarLint in my IDE about `typescript:S3735` (_"void" should not be used_) because the check does not seem to differentiate the type of the void argument 🙈 Should be fine for Promises in my opinion.